go/nomdl/parser_test.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2017 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package nomdl
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/attic-labs/noms/go/types"
+	"github.com/attic-labs/testify/suite"
+)
+
+type ParserSuite struct {
```

Comment:
You might consider https://blog.golang.org/subtests, which I've started using instead of testify suites. The benefit is mainly that you can actually run individual subtests via `go test`